modules/form-field/react/stories/stories_TextArea.tsx

```diff
@@ -14,143 +14,179 @@ const hintId = 'error-desc-id';
 storiesOf('Form Field/TextArea/Top Label', module)
   .addDecorator(withReadme(README))
   .add('Default', () => (
-    <FormField label="Label" inputId="textarea-plain">
-      {controlComponent(<TextArea />)}
-    </FormField>
+    <div className="story">
```

Comment:
You could update the `sectionDecorator` helper to remove the title (since we don't use those anymore now that we render the stories on the design site) and use that to avoid all this extra markup.